### PR TITLE
Add centralized support page

### PR DESCRIPTION
**Pull request by vikaspotluri123**
Needs all the reviews (engineering, design, copy)!

### CODE REVIEW

File: src/layouts/default.hbs [hunk 1]
```diff
@@ -115,10 +114,10 @@
 						<div class="site-footer-unit">
 							<div class="site-footer-unit-inner">
 								<div class="footer-top-links">
-									<div class="footer-title mb-16">Enterprise</div>
+									<div class="footer-title mb-16">Services</div>
```
Comment (vikaspotluri123) [inline src/layouts/default.hbs:117]: Not a fan of calling support a "service" - at this time it's free!

Comment (joshcosta) [reply src/layouts/default.hbs:117]: How about we replace the `Contact Us` email link with the link to the new support page, but keep the same name. And then enterprise can stay it's own section.

Comment (joshcosta) [reply src/layouts/default.hbs:117]: Actually wait, how about this:

ABOUT
- Blog
- Our Team
- Donate

POLICIES
- Terms of Service
- Privacy
- Security

GET IN TOUCH
- Enterprise Partnerships
- Support

**Comment by joshcosta** [comment]
The labels on the form need a little more contrast. Right now it's 3.03

**Comment by vikaspotluri123** [comment]
I think that's an issue with our color palette. I'll create a separate PR right after this with that fix.

File: scripts/support.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+if (search.get('type') === 'aggiecorps') {
+	/** @type {HTMLFormElement} */
+	const form = document.querySelector('form.support-form');
+	const prioritizer = document.createElement('input');
+	prioritizer.name = 'type';
+	prioritizer.type = 'hidden';
+	prioritizer.name = 'Corps Support'
+	form.insertBefore(prioritizer, form.children[0]);
+}
```
Comment (joshcosta) [inline scripts/support.js:19]: Nice!

**Comment by joshcosta** [comment]
> 

Yeah it's probably in many other places

File: src/layouts/default.hbs (hunk 1 above)
Comment (vikaspotluri123) [reply src/layouts/default.hbs:117]: I like it.

`Support` -> `Contact`?

Also, if you can think of any 1-word titles I think that would flow well.. Thoughts?

File: src/support.hbs [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+				</p>
+				<p>
+					<label for="subject" class="inline">What are you looking for?</label>
+					<select name="subject">
+						<option value="support">Get help using Gradebook</option>
+						<option value="bug_report">Report an issue</option>
+						<option value="feature_request">Suggest a feature</option>
+						<option value="partnership">Discuss a partnership</option>
+						<option value="other">Something else</option>
+					</select>
```
Comment (ramsays) [inline src/support.hbs:30]: Styling looks pretty standard (colors, border, etc.). Are you going to try to make this match our design language or just leave the default browser presentation?

**Review by ramsays — COMMENTED** [review]
Generally looks good, ditto on the contrast and just one comment on the select design!

File: src/support.hbs (hunk 3 above)
Comment (vikaspotluri123) [reply src/support.hbs:30]: Leaving it as-is for now is fine - I think vendors are working on a more styleable select (selectmenu) which we can switch to later

**Review by ramsays — APPROVED** [review]

**Comment by vikaspotluri123** [comment]
Is autocomplete working for ya'll? It's not for me 😕

File: src/layouts/default.hbs (hunk 1 above)
Comment (vikaspotluri123) [reply src/layouts/default.hbs]: Alright, we're good for final review.

File: src/support.hbs (hunk 3 above)
Comment (joshcosta) [inline src/support.hbs:30]: I think the select dropdown might look better on a new line

**Review by joshcosta — APPROVED** [review]
Autocomplete is working for me. 

While we're updating the footer, can you also update the copyright to 2022 if you get a chance?

**Comment by vikaspotluri123** [comment]
> While we're updating the footer, can you also update the copyright to 2022 if you get a chance?

[`15091de` (#165)](https://github.com/gradebook/marketing/pull/165/commits/15091de3f9144f378b20cff05bc5b8eaf3a8b3a1) 😄 There's an issue with the build pipeline where the layout doesn't get re-read in watch mode, so you have to restart the server

File: src/support.hbs (hunk 3 above)
Comment (vikaspotluri123) [reply src/support.hbs:30]: Main reason for keeping it on the same line is for the `Something else` - keeping the select on a separate line causes some grouping issues when it's visible

**Pull request merged**